model/intent/record_eeg.py

```diff
@@ -1,47 +1,50 @@
-                        default=0)
-    parser.add_argument('--ip-address', type=str, help='ip address', required=False, default='')
-    parser.add_argument('--serial-port', type=str, help='serial port', required=False, default='')
-    parser.add_argument('--mac-address', type=str, help='mac address', required=False, default='')
-    parser.add_argument('--other-info', type=str, help='other info', required=False, default='')
-    parser.add_argument('--serial-number', type=str, help='serial number', required=False, default='')
-    parser.add_argument('--file', type=str, help='file', required=False, default='')
-    parser.add_argument('--actions', type=int, help='number of actions to record',
-                        required=True)
-    parser.add_argument('--sessions', type=int, help='number of sessions per action to record',
-                    required=False, default=2)
-    # board id by name or id
-    parser.add_argument('--board-id', type=str, help='board id or name, check docs to get a list of supported boards',
-                        required=True)
+    parser.add_argument('--timeout',        type=int, required=False, default=0,  help='timeout for device discovery or connection')
+    parser.add_argument('--ip-port',        type=int, required=False, default=0,  help='ip port')
+    parser.add_argument('--ip-protocol',    type=int, required=False, default=0,  help='ip protocol, check IpProtocolType enum')
+    parser.add_argument('--ip-address',     type=str, required=False, default='', help='ip address')
+    parser.add_argument('--serial-port',    type=str, required=False, default='', help='serial port')
+    parser.add_argument('--mac-address',    type=str, required=False, default='', help='mac address')
+    parser.add_argument('--other-info',     type=str, required=False, default='', help='other info')
+    parser.add_argument('--serial-number',  type=str, required=False, default='', help='serial number')
+    parser.add_argument('--file',           type=str, required=False, default='', help='file',)
+    parser.add_argument('--actions',        type=int, required=True,              help='number of actions to record')
+    parser.add_argument('--sessions',       type=int, required=False, default=2,  help='number of sessions per action to record')
+    parser.add_argument('--window-length',  type=int, required=False, default=10, help='length in seconds of eeg data pulled per session')
+    parser.add_argument('--window-buffer',  type=int, required=False, default=2,  help='time in seconds before eeg data is recorded each session (delay after hitting enter)')
+    parser.add_argument('--overwrite',      type=int, required=False, default=1,  help='1 to overwrite/remove old recordings, 0 to add results as an additional data file')
+    parser.add_argument('--board-id',       type=str, required=True,              help='board id or name, check docs to get a list of supported boards. mu_02 is MUSE_2016_BOARD')
```

Comment:
Spacing in argument lists here is also not recommended, the problem here tends to be worse than in just variable assignments as you can have multiple variable columns in succession which need to be properly maintained.

Reply:
Yeah, you're right. Not worth the noise. If only python files were actually spreadsheets

Reply:
I've addressed all of these comments in recent commits, feel free to resolve if I didn't miss anything